src/linter/ui5Types/FixHintsGenerator.ts

```diff
@@ -0,0 +1,819 @@
+	["act.isActive", {
+		moduleName: "sap/ui/util/ActivityDetection", exportNameToBeUsed: "isActive",
+	}],
+	["initMobile", {
+		moduleName: "sap/ui/util/Mobile", exportNameToBeUsed: "init",
+	}],
+	["setIcons", {
+		moduleName: "sap/ui/util/Mobile", exportNameToBeUsed: "setIcons",
+	}],
+	["setMobileWebAppCapable", {
+		moduleName: "sap/ui/util/Mobile", exportNameToBeUsed: "setWebAppCapable",
+	}],
+	["storage", {
+		moduleName: "sap/ui/util/Storage", exportCodeToBeUsed: "new $moduleIdentifier($1, $2)",
+	}],
+	["getParseError", {
+		moduleName: "sap/ui/util/XMLHelper", exportNameToBeUsed: "getParseError",
+	}],
+	["parseXML", {
+		moduleName: "sap/ui/util/XMLHelper", exportNameToBeUsed: "parse",
+	}],
+	["serializeXML", {
+		moduleName: "sap/ui/util/XMLHelper", exportNameToBeUsed: "serialize",
+	}],
+
+	["device.is.standalone", {
+		exportCodeToBeUsed: "window.navigator.standalone",
+	}],
+	["support.retina", {
+		exportCodeToBeUsed: "window.devicePixelRatio >= 2",
+	}],
+	// TODO: Shall we add validation for a string?
+	["startsWith", {
+		exportCodeToBeUsed: "$1.startsWith($2)",
+	}],
+	// TODO: Shall we add validation for a string?
+	["startsWithIgnoreCase", {
+		exportCodeToBeUsed: "$1.toLowerCase().startsWith($2)",
+	}],
+	// TODO: Shall we add validation for a string?
+	["endsWith", {
+		exportCodeToBeUsed: "$1.endsWith($2)",
+	}],
+	// TODO: Shall we add validation for a string?
+	["endsWithIgnoreCase", {
+		exportCodeToBeUsed: "$1.toLowerCase().endsWith($2)",
+	}],
+	// TODO: Shall we add validation for a string?
+	// jquery guard: jQuery.sap.assert(typeof sPadChar === 'string' && sPadChar)
+	["padLeft", {
+		exportCodeToBeUsed: "$1.padStart($3, $2)",
+	}],
+	// TODO: Shall we add validation for a string?
+	// jquery guard: jQuery.sap.assert(typeof sPadChar === 'string' && sPadChar)
+	["padRight", {
+		exportCodeToBeUsed: "$1.padEnd($3, $2)",
+	}],
+	// TODO: $3 can be a function, so it should be $3.bind($2).
+	// Also inside of the jquery.sap.delayedCall if $3 is a string, then
+	// $2[$3].bind($2), so we always have the correct context
+	["delayedCall", {
+		exportCodeToBeUsed: "window.setTimeout($2[$3], $1, ...$4)",
+	}],
+	["clearDelayedCall", {
+		exportCodeToBeUsed: "window.clearTimeout($1)",
+	}],
+	// TODO: $3 can be a function, so it should be $3.bind($2).
+	// Also inside of the jquery.sap.intervalCall if $3 is a string, then
+	// $2[$3].bind($2), so we always have the correct context
+	["intervalCall", {
+		exportCodeToBeUsed: "window.setInterval($2[$3], $1, ...$4)",
+	}],
+	["clearIntervalCall", {
+		exportCodeToBeUsed: "window.clearInterval($1)",
+	}],
+	["domById", {
+		exportCodeToBeUsed: "window.document.getElementById($1)",
+	}],
+	["isEqualNode", {
+		exportCodeToBeUsed: "!!$1?.isEqualNode($2)",
+	}],
+	["newObject", {
+		// TODO: Decide on structuredClone or {...object}
+		// structuredClone can throw an exception in some cases
+		exportCodeToBeUsed: "structuredClone($1)",
```

Comment:
Docs suggest `Object.create`: https://ui5.sap.com/#/topic/a075ed88ef324261bca41813a6ac4a1c
The implementation uses `Object.create(oPrototype || null)`.

Reply:
Done